### ISSUE TITLE
Anchor link has aria-hidden is not output correctly

### ISSUE THREAD

**Issue by yamanoku** [nuxt/content#139]
I actually use it on my portfolio site. 👍 

### What is actually happening?

An anchor link is added in the heading with the a tag, and the attribute in it should be output as `aria-hidden`, but it is output as `ariaHidden`.

https://github.com/remarkjs/remark-autolink-headings/issues/49
`aria-hidden` has been added to this issue.

We're still trying to figure out what actually caused it, but we'd like to ask nuxt/content contributors to check this out.

### Version
@nuxt/content: v1.3.2

### Steps to reproduce
Browser （Stable） - Chrome, Safari, Firefox

<img width="1251" alt="Screen Shot 2020-06-14 at 18 59 40" src="https://user-images.githubusercontent.com/1996642/84591286-1df31580-ae78-11ea-9467-e56e4e4b48e5.png">

### What is Expected?
I want to render it the way it should be. `aria-hidden="true"`

<img width="1246" alt="Screen Shot 2020-06-14 at 19 00 00" src="https://user-images.githubusercontent.com/1996642/84591288-21869c80-ae78-11ea-9023-0a27328f3079.png">

ARIA Attribute is determined successfully.

**Comment by benjamincanac** [comment]
Hey @yamanoku,

Thanks for the bug report. It should be fixed in the next release.
